contracts/src/v0.8/dev/ocr2dr/OCR2DROracle.sol

```diff
@@ -7,27 +7,47 @@ import "../AuthorizedReceiver.sol";
 import "../../ConfirmedOwner.sol";
 
 /**
- * @title OCR2DR oracle contract (stub for now)
+ * @title OCR2DR oracle contract
  */
 contract OCR2DROracle is OCR2DROracleInterface, AuthorizedReceiver, ConfirmedOwner {
   event OracleRequest(bytes32 requestId, bytes data);
   event OracleResponse(bytes32 requestId);
 
+  error EmptyRequestData();
   error InvalidRequestID();
-  error Unauthorized();
-  error NonceMustBeUnique();
+  error LowGasForConsumer();
 
   struct Commitment {
     address client;
     uint256 subscriptionId;
   }
 
+  uint256 private constant MINIMUM_CONSUMER_GAS_LIMIT = 400000;
```

Comment:
Add Natspec comments for variables.

Reply:
Given the most variables are self-explanatory I decided to keep it simple as in many other contracts of us. Otherwise let me know if this is absolutely required.